opengrok-tools/src/main/python/opengrok_tools/utils/mirror.py

```diff
@@ -376,7 +376,17 @@ def check_project_configuration(multiple_project_config, hookdir=False,
     if not multiple_project_config:
         return True
 
+    logger.debug("Checking project configuration")
+
     for project_name, project_config in multiple_project_config.items():
+        logger.debug("Checking configuration of project {}".
+                     format(project_name))
+
+        if project_config is None:
+            logger.debug("Project {} has empty configuration".
```

Comment:
or maybe warning here would be more appropriate, leave it up to you

Reply:
good idea